OpenRA.Mods.HV/Traits/SupportPowers/TeleportPower.cs

```diff
@@ -56,6 +57,23 @@ sealed class TeleportPowerInfo : SupportPowerInfo
 		[Desc("Cursor to display when the targeted area is blocked.")]
 		public readonly string TargetBlockedCursor = "move-blocked";
 
+		[Desc("Maximum range from televator to be able to teleport.")]
+		public readonly int RangeFromTelevatorsSquared = 100;
+
+		public readonly int CellRangeFromTelevators = 10;
+
+		[Desc("Range circle color.")]
+		public readonly Color CircleColor = Color.FromArgb(128, Color.Red);
```

Comment:
```suggestion
		public readonly Color CircleColor = Color.FromArgb(128, Color.MediumSlateBlue);
```
maybe as the nuke is already red.